src/components/sections/Hero.section.tsx

```diff
@@ -52,7 +52,12 @@ const HeroSection: React.FC = () => {
                     REGISTER
                 </button>
                 <img
-                    className="-mt-2"
+                    style={{
+                        width: '100%',
+                        display: 'block',
+                        marginLeft: 'auto',
+                        marginRight: 'auto'
+                    }}
```

Comment:
```suggestion
                    className="w-full block mx-auto"
```

Just want to stick to Tailwind where ever possible.